### PR TITLE
@dzucconi - adds delta loader and initial interface for trending artists

### PR DESCRIPTION
**Pull request by broskoski**
`name` should probably be an enum, but I want to talk to @cavvia and get a list of all the supported method names here.

For reference: http://delta.artsy.net/?method=fetch&name=artist_follow&n=40

screen shot 2016-03-11 at 3 42 38 pm

### CODE REVIEW

File: lib/loaders/delta.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import qs from 'qs';
+import delta from '../apis/delta';
+import httpLoader from './http';
+
+export const deltaLoader = httpLoader(delta);
+
+export default (path, options = {}) => {
+  const queryString = qs.stringify(options, { arrayFormat: 'brackets' });
+  console.log('`${path}?${queryString}`', `${path}?${queryString}`)
```
Comment (dzucconi) [inline lib/loaders/delta.js]: Log

File: schema/node_definitions.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  switch (type) {
+    case 'Ping':
+      return gravity('system/ping');
+      break;
+    case 'Article':
+      return positron(`articles/${id}`);
+    default:
+      return null;
+  }
+});
```
Comment (dzucconi) [inline schema/node_definitions.js]: Accidental commit?

Comment (broskoski) [inline schema/node_definitions.js]: Oops, howd that get there.

**Comment by broskoski** [comment]
:green_heart:

**Comment by cavvia** [comment]
The method names supported are:

* artist_follow
* artist_inquiry
* artist_search
* artist_save
* artist_fair (num works in fairs)
* artist_auction_lot (cumulative price achieved at auction)

All the keys can have `_2t` appended to their name to fetch the top artists for each metric within double the base time period as well. Base time periods for each metric are:

```ruby {
               Delta::Event::ArtistFollow.key => 2.weeks,
               Delta::Event::ArtistSave.key => 1.month,
               Delta::Event::ArtistAuctionLot.key => 12.weeks,
               Delta::Event::ArtistFair.key => 12.weeks,
               Delta::Event::ArtistSearch.key => 2.weeks,
               Delta::Event::ArtistInquiry.key => 1.month }
```

**Pull request merged**